docs/content/guides/dagster/asset-versioning-and-caching.mdx

```diff
@@ -120,9 +135,11 @@ def multiplied_number(versioned_number):
     return versioned_number * 2
 ```
 
-Reload the location in dagit and you'll notice that `multiplied_number` is marked as "Never Materalized". Now click on the "Materialize" disclosure triangle and then "Materialize stale and missing" (the plain "Materialize" button ignores versioning) to materialize.
+In the Dagster UI, click **Reload definitions**. The `multipled_number` asset will be marked as **Never materialized**.
+
+Next, click the toggle to the right side of the **Materialize** button to display the **Materialize stale and missing** option. As the **Materialize** button ignores versioning, we need this option to ensure the `multipled_number` asset is properly materialized.
 
-Note that in the created run, _only_ the step associated with `multiplied_number` is run. The system knows that `versioned_number` is up to date and therefore can safely skip that computation. You can see this on the details page for the run:
+**Note**: In the created run, only the step associated with `multiplied_number` is run. The system knows that `versioned_number` is up to date and therefore can safely skip that computation. You can see this on the details page for the run:
```

Comment:
I think this is part of the main flow of text so would remove the "**Note**" here.